README.rst

```diff
@@ -2,3 +2,22 @@ thoth-station.github.io
 -----------------------
 
 Project Thoth website
+================
+
+How to contribute
+-----------------------
+Parts of the Thoth website are built using Jekyll via a `GitHub action <https://github.com/marketplace/actions/build-jekyll-for-github-pages>`_.
+Jekyll is `configured <https://github.com/thoth-station/thoth-station.github.io/blob/master/_config.yml>`_ to 
+read markdown files from the ` _j <https://github.com/thoth-station/thoth-station.github.io/tree/master/_j>`_ directory and render them in a Jekyll theme.
```

Comment:
```suggestion
read markdown files from the ` _j <https://github.com/thoth-station/thoth-station.github.io/tree/master/_j>`_ directory and render them in a Jekyll theme.
```
```suggestion
read markdown files from the `_j <https://github.com/thoth-station/thoth-station.github.io/tree/master/_j>`_ directory and render them in a Jekyll theme.
```

The extra space is causing rendering issue
can you please remove the extra space.
apologies if this came from my earlier review.